Makefile

```
@@ -4,6 +4,7 @@ help:
 	@echo 'make services          Run the services that `make dev` requires'
 	@echo "                       (Postgres) in Docker"
 	@echo 'make db                Upgrade the DB schema to the latest version'
+	@echo 'make alembic           Run alembic (DB migrations) commands.
```

```suggestion
	@echo 'make alembic           Run alembic (DB migrations) commands'
```

These docstrings don't have periods on the end, but mostly you need to add the `'` to fix a syntax error